torch/onnx/symbolic_helper.py

```diff
@@ -535,6 +656,14 @@ def _is_split_static(split_size_or_sizes, _outputs):
         return False
     return True
 
+def _optional_input_placeholder_tensor(g):
+    n = g.op("prim::Constant")
+    n.setType(OptionalType.ofTensor())
+    return n
+
+def _has_more_optional_parameters():
```

Comment:
nit: This function does not have much other use-cases.
Maybe just replace this with:
if _export_onnx_opset_version >= 13

Reply:
Just feel things like '_export_onnx_opset_version >= 13' lose readability.